dotCMS/src/main/java/com/dotcms/rest/DeleteBundlesByIdentifierForm.java

```diff
@@ -0,0 +1,35 @@
+package com.dotcms.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.List;
+
+@JsonDeserialize(builder = DeleteBundlesByIdentifierForm.Builder.class)
+public class DeleteBundlesByIdentifierForm {
+
+    private final List<String> identifiers;
+
+    private DeleteBundlesByIdentifierForm(final Builder builder) {
+
+        this.identifiers = builder.identifiers;
+    }
+
+    public List<String> getIdentifiers() {
+
+        return this.identifiers;
+    }
+
+    public static final class Builder {
+        @JsonProperty(required = true) private List<String> identifiers;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field identifiers has the same name as a method](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)